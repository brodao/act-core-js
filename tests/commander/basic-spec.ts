import { ACT } from '../../lib/index';

const { wrap } = require('@brodao/act-jest-snapshot-console');

test('Mensagem aninhada', () => {
	expect(
		wrap(() => {
			const cmd = ACT.initialzeApp({
				appInfo: {
					name: 'app_commander_test',
					version: '99.99.99',
					description: 'App Commander test',
					url: 'http://no.url',
				},
				commandText: true,
				optionsText: true,
			});

			return cmd.appCommand().helpInformation();
		})
	).toMatchSnapshot();
});
