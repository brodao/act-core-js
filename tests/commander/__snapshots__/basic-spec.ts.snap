// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comando HELP 1`] = `
Object {
  Symbol(act-jest-snapshot-raw): "Usage: app_commander_test <command> [options]

Options:
  -v, --version             output the version number
  --verbose                 detalha a execução (default: false)
  --no-banner               omite a abertura
  --log-to-file             gera arquivo com as ocorrências (default: false)
  --log-format <json|text>  formato do arquivo de logo (default: \\"text\\")
  -h, --help                display help for command
",
  Symbol(act-jest-snapshot-console): "",
}
`;
